.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 _esy/
+_build/
```

Comment:
opam? 😛

Reply:
I have no idea what generated the `_build/` dir tbh, but that sounds right to me 😂 lol